### PR TITLE
AB#39587 fix headers

### PR DESCRIPTION
**Pull request by lhao03**
Adds "Content-type" as key and "Application/json" as headers if the request type has a body.

### CODE REVIEW

File: tabs/src/components/app/views/runner/Request.jsx [hunk 1]
```diff
@@ -45,41 +48,54 @@ export function Request(props) {
...
     };
 
+    const deleteButton = () => <Button
+        tabIndex={-1}
+        icon={<TrashCanIcon className="button-icon" />}
+        circular
+        text
+        iconOnly
+        aria-label="delete"
+        title="Delete request header"
```
Comment (acchiang) [inline tabs/src/components/app/views/runner/Request.jsx]: please localize these!

**Review by acchiang — CHANGES_REQUESTED** [review]
small comment, don't forget translation :)

File: tabs/src/components/app/views/runner/Request.jsx (hunk 1 above)
Comment (lhao03) [reply tabs/src/components/app/views/runner/Request.jsx]: https://github.com/LokiLabs/microsoft-graph-explorer-teams-app/pull/57/commits/df064b88101315937b99551da1367cf746e76c9f

**Review by ayubun — APPROVED** [review]
LGTM! Tested locally ^^

File: tabs/src/components/app/views/runner/Request.jsx [hunk 2]
```diff
@@ -45,41 +48,54 @@ export function Request(props) {
         ],
     };
 
+    const deleteButton = () => <Button
+        tabIndex={-1}
+        icon={<TrashCanIcon className="button-icon" />}
+        circular
+        text
+        iconOnly
+        aria-label="delete"
```
Comment (acchiang) [inline tabs/src/components/app/views/runner/Request.jsx]: the aria label needs to be localized as well!

**Review by acchiang — DISMISSED** [review]

**Review by omersayshi — APPROVED** [review]

File: tabs/src/components/app/views/runner/Request.jsx (hunk 2 above)
Comment (lhao03) [reply tabs/src/components/app/views/runner/Request.jsx]: https://github.com/LokiLabs/microsoft-graph-explorer-teams-app/pull/57/commits/dae8f359fc4851554f2f48601fa647dcb83de88b

**Pull request merged**